static/app/components/actions/ignore.tsx

```diff
@@ -36,6 +39,7 @@ type Props = {
 };
 
 const IgnoreActions = ({
+  group,
```

Comment:
Don't think it makes sense to pass `group` into this component since it is also used on the issue stream where it has to check against multiple groups. Could make it `groups` or `issueCategories` instead?

Reply:
Agreed @malwilley, I noticed this a bit later and I accounted for this case in my most recent PR, although the implementation is a bit gross 😬 

I'm checking in the issue stream whether a Performance Issue is selected, and then I pass in a fake group object with the `issueCategory` property set to performance.

I was initially going to have the disabled state controlled by the consumers of the component as you mentioned, but yeah the code felt overly verbose for what we were trying to achieve